### PR TITLE
docs-fr | setup | custom-cloud coreos

### PR DESCRIPTION
**Pull request by Smana**

### CODE REVIEW

**Comment by perriea** [comment]
/hold
@lledru an opinion ? :)

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    Configure un cluster de 3 machines sur Vagrant, le réseau du cluster étant fourni par Weave.
+
+* [**Cluster multi-noeuds en utilisant cloud-config et Vagrant**](https://github.com/pires/kubernetes-vagrant-coreos-cluster/blob/master/README.md)
+
+    Configure un cluster local composé d'un master et de plusieurs workers sur l'hyperviseur de votre choix: VirtualBox, Parallels, or VMware
+
+* [**Cluster d'un seul noeud en utilisant une petite application macOS**](https://github.com/rimusz/kube-solo-osx/blob/master/README.md)
+
+    Guide permettant d'obtenir un cluster d'un seul noeud faisant office de master et worker et controller par une application macOS menubar.
```
Comment (etiennecoutaud) [inline content/fr/docs/setup/custom-cloud/coreos.md]: Is it `contrôler` verb ? in that case `s/controller/contrôlé`

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    (basé sur xhyve et CoreOS)
+
+* [**Cluster multi-noeuds avec Vagrant et fleet en utilisant une petite application macOS**](https://github.com/rimusz/coreos-osx-gui-kubernetes-cluster/blob/master/README.md)
+
+    Guide permettant d'obtenir un cluster composé d'un master, de plusieurs workers controller par une application macOS menubar.
```
Comment (etiennecoutaud) [inline content/fr/docs/setup/custom-cloud/coreos.md]: same with `controller`

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+* [**Cluster multi-noeuds en utilisant cloud-config et Weave sur Vagrant**](https://github.com/errordeveloper/weave-demos/blob/master/poseidon/README.md)
+
+    Configure un cluster de 3 machines sur Vagrant, le réseau du cluster étant fourni par Weave.
+
+* [**Cluster multi-noeuds en utilisant cloud-config et Vagrant**](https://github.com/pires/kubernetes-vagrant-coreos-cluster/blob/master/README.md)
+
+    Configure un cluster local composé d'un master et de plusieurs workers sur l'hyperviseur de votre choix: VirtualBox, Parallels, or VMware
+
+* [**Cluster d'un seul noeud en utilisant une petite application macOS**](https://github.com/rimusz/kube-solo-osx/blob/master/README.md)
```
[the remark this answers is not in the document]
Comment (Smana) [reply content/fr/docs/setup/custom-cloud/coreos.md]: agree

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+    Guide permettant d'obtenir un cluster d'un seul noeud faisant office de master et worker et controller par une application macOS menubar.
+    (basé sur xhyve et CoreOS)
+
+* [**Cluster multi-noeuds avec Vagrant et fleet en utilisant une petite application macOS**](https://github.com/rimusz/coreos-osx-gui-kubernetes-cluster/blob/master/README.md)
```
[the remark this answers is not in the document]
Comment (Smana) [reply content/fr/docs/setup/custom-cloud/coreos.md]: +1

File: content/fr/docs/setup/custom-cloud/coreos.md (hunk 1 above)
Comment (Smana) [reply content/fr/docs/setup/custom-cloud/coreos.md]: +1

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    Guide pour l'installation d'un cluster multi-noeuds sur Vagrant.
+    L'outil de déploiement permet de configurer indépendemment le nombre de noeuds etcd, masters et workers afin d'obtenir un control plane en haute disponibilité.
+
+* [**Noeud unique sur Vagrant**](https://coreos.com/kubernetes/docs/latest/kubernetes-on-vagrant-single.html)
+
+    C'est la façon la plus rapide d'installer un environnement de développement local Kubernetes.
+    Il suffit simplement de `git clone`, `vagrant up` puis configurer `kubectl`.
+
+
+* [**Guide complaet pas à pas**](https://coreos.com/kubernetes/docs/latest/getting-started.html)
```
[the remark this answers is not in the document]
Comment (Smana) [reply content/fr/docs/setup/custom-cloud/coreos.md]: good catch

**Comment by Smana** [comment]
@etiennecoutaud i've applied the changes. Let me know if that's ok for you so I can squash the commits

**Comment by etiennecoutaud** [comment]
LGTM

File: content/fr/docs/setup/custom-cloud/coreos.md (hunk 2 above)
Comment (lledru) [inline content/fr/docs/setup/custom-cloud/coreos.md]: typo: controllé
menubar -> is this correct?

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+## Niveau de support
+
+
+| IaaS Provider | Config. Mgmt | OS     | Networking | Docs                                        | Conforms | Support Level                                                                                          |
+| ------------- | ------------ | ------ | ---------- | ------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------ |
+| GCE           | CoreOS       | CoreOS | flannel    | [docs](/docs/getting-started-guides/coreos) |          | Community ([@pires](https://github.com/pires))                                                         |
+| Vagrant       | CoreOS       | CoreOS | flannel    | [docs](/docs/getting-started-guides/coreos) |          | Community ([@pires](https://github.com/pires), [@AntonioMeireles](https://github.com/AntonioMeireles)) |
+
+Pour le niveaud de support de toutes les solutions se référer au [Tableau des solutions](/docs/getting-started-guides/#table-of-solutions).
```
Comment (lledru) [inline content/fr/docs/setup/custom-cloud/coreos.md]: typo: niveau

File: content/fr/docs/setup/custom-cloud/coreos.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    Configure un cluster local composé d'un master et de plusieurs workers sur l'hyperviseur de votre choix: VirtualBox, Parallels, or VMware
+
+* [**Cluster d'un seul noeud en utilisant une application macOS**](https://github.com/rimusz/kube-solo-osx/blob/master/README.md)
+
+    Guide permettant d'obtenir un cluster d'un seul noeud faisant office de master et worker et contrôlé par une application macOS menubar.
+    (basé sur xhyve et CoreOS)
+
+* [**Cluster multi-noeuds avec Vagrant et fleet en utilisant une petite application macOS**](https://github.com/rimusz/coreos-osx-gui-kubernetes-cluster/blob/master/README.md)
+
+    Guide permettant d'obtenir un cluster composé d'un master, de plusieurs workers contrôlé par une application macOS menubar.
```
Comment (oussemos) [inline content/fr/docs/setup/custom-cloud/coreos.md]:
```suggestion
    Guide permettant d'obtenir un cluster composé d'un master, de plusieurs workers contrôlés par une application macOS menubar.
```

**Comment by Smana** [comment]
@oussemos @lledru, I've applied the changes

**Comment by perriea** [comment]
/lgtm
ahah @Smana 🤣

**Pull request merged**